src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs

```diff
@@ -67,7 +76,10 @@ public void Initialize()
 #if ANDROID || IOS || MACCATALYST
 		Drawable = new DrawingViewDrawable(this);
 #elif WINDOWS
-		((Microsoft.Maui.Graphics.Win2D.W2DGraphicsView)Content).Drawable = new DrawingViewDrawable(this);
+		if (OperatingSystem.IsWindowsVersionAtLeast(10, 0, 18362))
```

Comment:
What will happen if the windows version is below that?

Reply:
we do not see the lines:
![image](https://user-images.githubusercontent.com/33021114/164548213-ede600e5-2613-4447-a25b-75d3cc8095bc.png)

we see image and events, but DrawingView Drawable is empty (but control still take it's place)

Reply:
invisible ink feature

Reply:
can we add a log message informing the user that he needs to increase the target version?

Reply:
We could use the `SupportedOSPlatformAttribute` and specify the version number. 

Something like this ensures that a warning appears for all versions of windows older than 10.0.18362.0:
```cs
[UnsupportedOSPlatform("windows")]
[SupportedOSPlatform("windows10.0.18362.0")
```

https://docs.microsoft.com/en-us/dotnet/standard/analyzers/platform-compat-analyzer#advanced-scenarios-for-attribute-combinations

Reply:
I would prefer a simple log message. SupportedOSPlatform attribute generates a warning. Also, it is a MauiDrawingView, which is not used directly in Xaml or C#